core/src/main/scala/org/dbpedia/extraction/wikiparser/impl/json/JsonWikiParser.scala

```diff
@@ -35,22 +30,32 @@ object JsonWikiParser {
 
 class JsonWikiParser {
 
-  def apply(page : WikiPage) : Option[JsonNode] =
-  {
-    if (page.format == null || page.format != "application/json")
-    {
+  def apply(page: WikiPage): Option[JsonNode] = {
+    if (page.format == null || page.format != "application/json") {
       None
     }
-    else
-    {
-      val IntRegEx = new Regex("(\\d+)")
-      val jsonObject : JSONObject  = new JSONObject(page.source)
-      val jsonConverter = new JsonConverter("http://data.dbpedia.org/resource", new DataObjectFactoryImpl())
-      val Some(title) = IntRegEx findFirstIn page.title.toString()
-      val itemDocument : ItemDocument = jsonConverter.convertToItemDocument(jsonObject, "Q"+title)
-      Some(new JsonNode(page,itemDocument))
+    else {
+
+      val mapper = new ObjectMapper()
+      val jsonString = fixBrokenJson(page.source)
```

Comment:
@alismayilov , Can you move this inside the catch? only ~20% of the pages have errors atm and this will make parsing faster in most cases